configure.ac

```diff
@@ -70,35 +66,40 @@ case $host_os in
    ;;
 esac
 
-CFLAGS="-W $CFLAGS"
```

Comment:
Maybe document the removing of `-W` option?

Reply:
Apparently I've documented it here already https://github.com/bitcoin-core/secp256k1/pull/944/files#diff-49473dca262eeab3b4a43002adb08b4db31020d190caaad1594b47f1d5daa810R95

Reply:
Ah, right.